app/controllers/welcome_controller.rb

```diff
@@ -1,5 +1,7 @@
 class WelcomeController < ApplicationController
   def index
-    redirect_to event_path(Event.first)
+    if Event.first
+      redirect_to event_path(Event.first)
```

Comment:
`redirect_to Event.first if Event.first` works as well, but you're still querying ´Event.first` twice.